tools/pytorch2onnx.py

```diff
@@ -12,12 +13,28 @@
 from mmedit.models import build_model
 
 
+def show_result_pyplot(img, title='', block=True):
```

Comment:
So this function simply shows an image? Can you rewrite it using cv2.imshow() or imwrite?